tmva/sofie/inc/TMVA/OperatorList.hxx

```diff
@@ -27,4 +27,4 @@
 #include "TMVA/ROperator_LayerNormalization.hxx"
 #include "TMVA/ROperator_Expand.hxx"
 #include "TMVA/ROperator_Gather.hxx"
-#include "TMVA/ROperator_Swish.hxx"
+#include "TMVA/ROperator_Swish.hxx"
```

Comment:
Same, Swish operator was added from another PR, I believe.

Reply:
Okay, this is required for fixing issue mentioned here
https://github.com/root-project/root/pull/12945#issuecomment-1592685977